### ISSUE TITLE
couldn't find file 'codemirror' with type 'application/javascript'

### ISSUE THREAD

**Issue by rhabacker** [openSUSE/open-build-service#7716]
With obs server from virtual box image (obs-server.x86_64-2.9.6-Build1.1.vdi) precompiling assets fails with the following error: 
rails aborted!
Sprockets::FileNotFound: couldn't find file 'codemirror' with type 'application/javascript'
Checked in these paths: 
  /srv/www/obs/api/app/assets/icons
  /srv/www/obs/api/app/assets/images
  /srv/www/obs/api/app/assets/javascripts
  /srv/www/obs/api/app/assets/stylesheets
  /usr/lib64/ruby/gems/2.5.0/gems/peek-dalli-1.2.0/app/assets/javascripts
  /usr/lib64/ruby/gems/2.5.0/gems/peek-1.0.1/app/assets/javascripts
  /usr/lib64/ruby/gems/2.5.0/gems/peek-1.0.1/app/assets/stylesheets
  /usr/lib64/ruby/gems/2.5.0/gems/mousetrap-rails-1.4.6/app/assets/stylesheets
  /usr/lib64/ruby/gems/2.5.0/gems/mousetrap-rails-1.4.6/vendor/assets/javascripts
  /usr/lib64/ruby/gems/2.5.0/gems/coffee-rails-4.2.2/lib/assets/javascripts
  /usr/lib64/ruby/gems/2.5.0/gems/flot-rails-0.0.7/vendor/assets/javascripts
  /usr/lib64/ruby/gems/2.5.0/gems/cocoon-1.2.11/app/assets/javascripts
  /usr/lib64/ruby/gems/2.5.0/gems/actioncable-5.1.4/lib/assets/compiled
  /usr/lib64/ruby/gems/2.5.0/gems/actionview-5.1.4/lib/assets/compiled
/srv/www/obs/api/app/assets/javascripts/webui/application/cm2/index.js:13
bin/rails:4:in `require'
bin/rails:4:in `<main>'
Tasks: TOP => assets:precompile
(See full trace by running task with --trace)

Expected Result
============
The assets should be precompiled and not error should occur

How to Reproduce
===============
1. download virtual box image of obs server from http://download.opensuse.org/repositories/OBS:/Server:/2.9/images/obs-server.x86_64.vdi
2. configure virtualbox machine and start it
3. login into virtual machine
4. run `/srv/www/obs/api`
5. run `RAILS_ENV=production bundle exec rails.ruby2.5 assets:precompile`

**Comment by coolo** [comment]
the assets should already be precompiled in said image. If you want to redo them, it's IMO up to you to install the dependencies required to do so.

Or is this part of some manual to do this?

**Comment by hennevogel** [comment]
Yes we specifically precompile them during build and bundle without assets (see `.bundle/config`) . You would have to reinstall the bundle.
